tests/ex09_toml/store-data.toml

```diff
@@ -0,0 +1,23 @@
+name = "Acme store"
+id = 982
+created_on = "12/21/2017 14:21:55"
+data_from = "2017-12-18T00:00:00"
+data_to = "2017-12-19T23:59:59"
+price = "98237.448"
+[[days]]
+date = "2017-12-18"
+logged_on = "19:20"
+open_at = "08:00:00"
+closed_on = "19:00:00"
+customers = 487
+day_type = "Normal"
+sales = 27223.65
+
+[[days]]
+date = "2017-12-19"
+logged_on = "17:50"
+open_at = "10:30:00"
+closed_on = "17:30:00"
+customers = 192
+day_type = "Holiday"
+
```

Comment:
To the best of my knowledge, TOML also supports the following data types that are from missing from this test.
Boolean
Inline Table
Vanilla Arrays
Nested Arrays of tables

Reply:
We reviewed your PR as a training exercise. We recognize this was meant to be a minimal TOML add on and to that end it looks awesome! Congrats!